spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/core/PubSubException.java

```diff
@@ -20,7 +20,6 @@
 
 /**
  * The Spring Google Cloud Pub/Sub specific {@link NestedRuntimeException}.
- *
```

Comment:
That's not correct: the `@author` has to start from its own paragraph, over blank line.

Anyway I asked about removing blank lines in the *method Javadocs*, not class level...

Reply:
@artembilan Understood. Will add blank line in the class Javadoc.